common/unsafe/src/main/java/org/apache/spark/unsafe/types/CalendarInterval.java

```diff
@@ -27,8 +29,21 @@
 import static org.apache.spark.sql.catalyst.util.DateTimeConstants.*;
 
 /**
- * The internal representation of interval type.
+ * The data type representing calendar intervals. The calendar interval is stored internally in
```

Comment:
data type? how about `The class representing ...`

Reply:
ok